main.cc

```diff
@@ -1452,6 +1452,10 @@ int main(int ac, char** av) {
                 sl_controller.stop().get();
             });
 
+            auto stop_sl_updates = defer_verbose_shutdown("service level controller updates", [] {
+                sl_controller.local().drain().get();
+            });
+
```

Comment:
Does it make sense to have separate action for `s_l_controller::drain` now? It's immediately followed by `sharded<s_l_controller>::stop`, so if you remove this draining defer, seastar will call `s_l_controller::stop` automagically and this `stop` already calls `s_l_controller::drain` inside.

Reply:
Yeah, this deferrer can be reduced now